packit/config/aliases.py

```diff
@@ -138,16 +138,27 @@ def get_valid_build_targets(*name: str, default: str = DEFAULT_VERSION) -> set:
 
 
 def get_branches(
-    *name: str, default=DEFAULT_VERSION, default_dg_branch="main"
+    *name: str,
+    default: str = DEFAULT_VERSION,
+    default_dg_branch: str = "main",
+    for_event: bool = False,
```

Comment:
Can we name it more explicitely? Something like `add_aliases`? The `for_event` name sadly does not say anything without checking the docstring.

Reply:
switched to `with_aliases`